src/App.tsx

```diff
@@ -20,16 +35,20 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                value={query}
+                onChange={event => (
+                  setQuery(event.target.value)
+                )}
```

Comment:
Would be nice to create event handler function instead of setting state here, but in this case it's not critical